sample-apps/donet-sample-app/donet-sample-app/Controllers/AppController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Amazon.S3;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net.Http;
+
+namespace donet_sample_app.Controllers
+{
+
+    var meter = new Meter("MyApplication");
+
+    var counter = meter.CreateCounter<int>("Requests");
+    var histogram = meter.CreateHistogram<float>("RequestDuration", unit: "ms");
+
+    [ApiController]
+    [Route("[controller]")]
+    public class AppController : ControllerBase
+    {
+        private readonly AmazonS3Client s3Client = new AmazonS3Client();
+        private readonly HttpClient httpClient = new HttpClient();
+
+        [HttpGet]
+        [Route("/outgoing-http-call")]
+        public string OutgoingHttp()
+        {
+            counter.Add(1, KeyValuePair.Create<string, object?>("name", name));
+            //var stopwatch = Stopwatch.StartNew();
```

Comment:
```suggestion
```

Reply:
Introduced Stopwatch again for Random metrics